doc/releases/migration-guide-4.0.rst

```diff
@@ -156,6 +156,17 @@ Bluetooth Audio
   This needs to be added to all instances of VCP Volume Renderer callback functions defined.
   (:github:`76992`)
 
+* The Unicast Server has a new registration function :code:`bt_bap_unicast_server_register` which
+  takes a :code:`struct bt_bap_unicast_server_register_param` as argument. This allows the Unicast
+  Server to dynamically register Source and Sink ASE count at runtime. The old
+  :code:`CONFIG_BT_ASCS_ASE_SRC_COUNT` and :code:`CONFIG_BT_ASCS_ASE_SNK_COUNT` has been renamed to
```

Comment:
Suggest to also add the new names of `CONFIG_BT_ASCS_ASE_SRC_COUNT` and `CONFIG_BT_ASCS_ASE_SNK_COUNT` here.

When we refer to Kconfig options we usually use

```
:kconfig:option:
```
instead of 
```
:code:
```